tox.ini

```diff
@@ -0,0 +1,11 @@
+# Tox (http://tox.testrun.org/) is a tool for running tests
+# in multiple virtualenvs. This configuration file will run the
+# test suite on all supported python versions. To use it, "pip install tox"
+# and then run "tox" from this directory.
+
+[tox]
+envlist=py27
+
+[testenv]
+deps=-rrequirements.txt
```

Comment:
The tox docs say to use `-rrequirements.txt`, I'm assuming because it means `-r requirements.txt`